pkg/ebpf/c/common/buffer.h

```diff
@@ -339,102 +339,116 @@ statfunc int save_sockaddr_to_buf(args_buffer_t *buf, struct socket *sock, u8 in
 
 #define DEC_ARG(n, enc_arg) ((enc_arg >> (8 * n)) & 0xFF)
 
+#define BITMASK_INDIRECT_VALUE_TYPES                                                               \
+    ((u64) 1 << STR_T | (u64) 1 << SOCKADDR_T | (u64) 1 << INT_ARR_2_T | (u64) 1 << TIMESPEC_T)
+
+#define BITMASK_COMMON_TYPES                                                                       \
+    ((u64) 1 << INT_T | (u64) 1 << UINT_T | (u64) 1 << LONG_T | (u64) 1 << ULONG_T |               \
+     (u64) 1 << OFF_T_T | (u64) 1 << MODE_T_T | (u64) 1 << DEV_T_T | (u64) 1 << SIZE_T_T |         \
+     (u64) 1 << POINTER_T | (u64) 1 << STR_ARR_T | (u64) 1 << BYTES_T | (u64) 1 << U16_T |         \
+     (u64) 1 << CRED_T | (u64) 1 << UINT64_ARR_T | (u64) 1 << U8_T)
+
+#define ARG_TYPE_MAX_ARRAY (u8) TIMESPEC_T // last element defined in argument_type_e
```

Comment:
As we discuss, maybe `TIMESPEC_T` should be `TYPE_MAX` in order to avoid the mistake of adding a field in the struct `argument_type_e` and forget to change here. One suggestion is add the following comment in the struct `argument_type_e`:
```
// Important: If any more fields are added to this enum, the array 
// type_size_table (and related defines) must be updated accordingly.
```

Reply:
Tks, @rscampos. I'm going to add such comment.